calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/CalligraphyFactory.java

```diff
@@ -114,7 +114,10 @@ public View onViewCreated(View view, Context context, AttributeSet attrs) {
     }
 
     void onViewCreatedInternal(View view, final Context context, AttributeSet attrs) {
-        if (view instanceof TextView) {
+        if (!CalligraphyConfig.get().isIgnoredClass(view.getClass()) &&
```

Comment:
isIgnoredClass should take the current view and the Configuration separately as method arguments.